server/internal/service/bot.go

```diff
@@ -68,8 +68,8 @@ func (svc *BotService) CreateBot(cid string, params *requestparams.CreateBotPara
 		BotPort:     params.BotPort,
 		BotToken:    params.BotToken,
 		Nickname:    params.Nickname,
-		Avatar:      params.Avatar,
-		Description: params.Description,
+		Avatar:      *params.Avatar,
+		Description: *params.Description,
```

Comment:
why only these two fields using *?

Reply:
These param are optional, so use pointer to get empty string params.